docs/how-to-use-customized-configuration.md

```diff
@@ -1,294 +0,0 @@
-# How to use a customized properties file with Scalar Helm Charts
-
-You can set your customized properties files with Scalar Helm Charts (ScalarDB, ScalarDL Ledger, ScalarDL Auditor, and ScalarDL Schema Loader). Also, you can specify some flags in addition to the customized properties file with the ScalarDL Schema Loader chart. This document explains how to set properties files and flags with Scalar Helm Charts.
-
-## Set your properties file to a custom values file
```

Comment:
I moved this section (how to set `*.properties`) to each document under [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md).